splash/lua_modules/splash.lua

```diff
@@ -132,5 +132,21 @@ function Splash:jsfunc(...)
   return unwraps_errors(func)
 end
 
+function Splash:__newindex( index, value )
+  if index == "js_enabled" then
+    self.members[index] = value
+    self:set_js_enabled(value)
+  else
+    rawset(Splash, index, value)
+  end
+end
+
+function Splash:__index(index)
```

Comment:
`Splash.__index` was assigned to another value above. It is better to delete that assignment if you're changing what's `Splash.__index`.